deployment/playbooks/base-install.yml

```diff
@@ -0,0 +1,211 @@
+---
+- import_playbook: includes/ensure_host_keys.yml
+
+- name: environmental setup
+  hosts: all
+  tasks:
+    - include_tasks: includes/defaults.yml
+
+- name: base system install
+  hosts: all
+  vars:
+    # Ansible will use this path on remote hosts to run the interpreter with.
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    #- include_tasks: includes/common.yml
+
+    - name: Initialize git variables for downloading repository without git
+      set_fact:
+        git_branch: remote-platforms
+        git_organization: craig8
```

Comment:
This should point to 'volttron' or be made configurable

Reply:
yup